test/unit/transpiler/passes/scheduling/test_dynamical_decoupling.py

```diff
@@ -474,41 +473,6 @@ def test_insert_dd_bad_sequence(self):
         with self.assertRaises(TranspilerError):
             pm.run(self.ghz4)
 
-    @data(0.5, 1.5)
-    def test_dd_with_calibrations_with_parameters(self, param_value):
-        """Check that calibrations in a circuit with parameters work fine."""
-
-        circ = QuantumCircuit(2)
-        circ.x(0)
-        circ.cx(0, 1)
-        circ.rx(param_value, 1)
-
-        rx_duration = int(param_value * 1000)
-
-        with pulse.build() as rx:
-            pulse.play(
-                pulse.Gaussian(rx_duration, 0.1, rx_duration // 4),
-                pulse.DriveChannel(1),
-            )
-
-        circ.add_calibration("rx", (1,), rx, params=[param_value])
-
-        durations = DynamicCircuitInstructionDurations([("x", None, 100), ("cx", None, 300)])
-
-        dd_sequence = [XGate(), XGate()]
-        pm = PassManager(
-            [
-                ASAPScheduleAnalysis(durations),
-                PadDynamicalDecoupling(durations, dd_sequence, schedule_idle_qubits=True),
-            ]
-        )
-        dd_circuit = pm.run(circ)
-
-        for instruction in dd_circuit.data:
-            op = instruction.operation
-            if isinstance(op, RXGate):
-                self.assertEqual(op.duration, rx_duration)
-
```

Comment:
I believe this is failing because `duration` is deprecated?

Reply:
the test assertion now matches the test name!